src/Maestro/Microsoft.AspNetCore.ApiVersioning.Swashbuckle/swagger.json

```diff
@@ -0,0 +1,3649 @@
+{
```

Comment:
Remove this file please. I added it to my change by accident.

Reply:
Done.